app/views/shared/_comment.html.erb

```diff
@@ -0,0 +1,4 @@
+<div class="comment">
```

Comment:
- [ ] As the previous reviewer suggested that you don't have `views` files for comments. 
P.S Yes, You should have a comments folder in your views files `(./app/views/comments/new.html.erb)` so your project can be approved 
`There should be a form in the views/comments/new.html.erb file which accepts input for creating comments.`